src/main/java/ru/practicum/shareit/user/storage/InMemoryUserStorage.java

```diff
@@ -0,0 +1,57 @@
+package ru.practicum.shareit.user.storage;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import ru.practicum.shareit.IdCreator;
+import ru.practicum.shareit.user.dto.UserDto;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class InMemoryUserStorage implements UserStorage {
+
+    private final IdCreator idCreator = new IdCreator();
+    private final Map<Integer, UserDto> users = new HashMap<>();
```

Comment:
Соответственно тут тоже
```suggestion
    private final Map<Integer, User> users = new HashMap<>();
```